delfin/tests/unit/api/v1/test_storages.py

```diff
@@ -44,24 +34,8 @@ def setUp(self):
 
     @mock.patch.object(db, 'storage_get',
                        mock.Mock(return_value={'id': 'fake_id'}))
-    @mock.patch('delfin.common.config.load_json_file')
-    def test_delete(self, mock_load_json_file):
-        req = fakes.HTTPRequest.blank('/storages/fake_id')
-        mock_load_json_file.return_value = fake_schedular_config
```

Comment:
this test need update

Reply:
The rest case is not deleted. Only testcase ```test_delete_invalid_schedular_conf```` is deleted